### DISCUSSION TITLE
atops cli not installing

### DISCUSSION THREAD

**Discussion by Aman1919** [aptos-labs/aptos-developer-discussions#294] [Questions] [answered]
### Discord user ID

aman.sgh

### Describe your question in detail.

i am trying to install aptos cli , so i followed the docs but after entering the cmd it was doing nothing 

### What error, if any, are you getting?

![image](https://github.com/aptos-labs/aptos-developer-discussions/assets/89124765/d6747d27-98b9-42d3-8b85-25217228ec35)

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

Python SDK

**Comment by gregnazario** [comment, marked as answer]
You should be able to add the CLI to your path first https://aptos.dev/en/build/cli/install-cli/install-cli-windows#copy-and-run-the-command-to-update-your-path-from-the-terminal

Then, try the `aptos` command again

**Reply by Aman1919** [reply]
Ok thanks . Got it
